en/docs/develop-components/ballerina-private-package-usage.md

```diff
@@ -0,0 +1,34 @@
+# Ballerina Private Package Usage
```

Comment:
@shaniR I searched this in https://wso2.com/choreo/docs/ and could not find a topic called "Sharing and reusing services". Can you point me to the location?

Reply:
Yes what I meant is, there's a topic we have listed in our structure as "Sharing and resuing services". If you could name this doc the same it would be fine.

Reply:
@Mirage20  shall we rename this document to "Sharing and Reusing Services"